queries/python/highlights.scm

```diff
@@ -0,0 +1,150 @@
+;; From tree-sitter-python licensed under MIT License
+; Copyright (c) 2016 Max Brunsfeld
+
+; Identifier naming conventions
+
+;((identifier) @constructor
+ ;(#match? @constructor "^[A-Z]"))
+
+;((identifier) @constant
+ ;(#match? @constant "^[A-Z][A-Z_]*$"))
+
+; Function calls
+
+(decorator) @function
+
+(call
+  function: (attribute attribute: (identifier) @function.method))
+(call
+  function: (identifier) @function)
+
+;; Builtin functions
+
+;((call
+  ;function: (identifier) @function.builtin)
+ ;(#match?
+   ;@function.builtin
+   ;"^(abs|all|any|ascii|bin|bool|breakpoint|bytearray|bytes|callable|chr|classmethod|compile|complex|delattr|dict|dir|divmod|enumerate|eval|exec|filter|float|format|frozenset|getattr|globals|hasattr|hash|help|hex|id|input|int|isinstance|issubclass|iter|len|list|locals|map|max|memoryview|min|next|object|oct|open|ord|pow|print|property|range|repr|reversed|round|set|setattr|slice|sorted|staticmethod|str|sum|super|tuple|type|vars|zip|__import__)$"))
+
+;; Function definitions
+
+(function_definition
+  name: (identifier) @function)
+
+(identifier) @variable
+(attribute attribute: (identifier) @property)
+(type (identifier) @type)
+(parameters
+  (identifier) @parameter) 
+
+; Literals
+
+(none) @constant.builtin
+(true) @constant.builtin
+(false) @constant.builtin
+
+(integer) @number
+(float) @number
+
+(comment) @comment
+(string) @string
+(escape_sequence) @escape
+
+(interpolation
+  "{" @punctuation.special
+  "}" @punctuation.special) @embedded
+
+; Tokens
+
+"-" @operator
+"-=" @operator
+"!=" @operator
+"*" @operator
+"**" @operator
+"**=" @operator
+"*=" @operator
+"/" @operator
+"//" @operator
+"//=" @operator
+"/=" @operator
+"&" @operator
+"%" @operator
+"%=" @operator
+"^" @operator
+"+" @operator
+"+=" @operator
+"<" @operator
+"<<" @operator
+"<=" @operator
+"<>" @operator
+"=" @operator
+"==" @operator
+">" @operator
+">=" @operator
+">>" @operator
+"|" @operator
+"~" @operator
+"and" @operator
+"in" @operator
+"is" @operator
+"not" @operator
+"or" @operator
+
+; Keywords
+
+"as" @keyword
+"assert" @keyword
+"async" @keyword
+"await" @keyword
+"break" @keyword
+"class" @keyword
+"continue" @keyword
+"def" @keyword
+"del" @keyword
+"elif" @keyword
+"else" @keyword
+"except" @keyword
+"exec" @keyword
+"finally" @keyword
+"for" @keyword
+"from" @keyword
+"global" @keyword
+"if" @keyword
+"import" @keyword
+"lambda" @keyword
+"nonlocal" @keyword
+"pass" @keyword
+"print" @keyword
+"raise" @keyword
+"return" @keyword
+"try" @keyword
+"while" @keyword
+"with" @keyword
+"yield" @keyword
+
+; Additions for nvim-treesitter
+"(" @punctuation.bracket
+")" @punctuation.bracket
+"[" @punctuation.bracket
+"]" @punctuation.bracket
+
+"," @punctuation.delimiter
+"." @punctuation.delimiter
```

Comment:
Don't you think that we can also add `:` as a delimiter ?

Reply:
`;` is at least not. Have to look up what it is.